src/test-run/command/test-done.js

```diff
@@ -0,0 +1,8 @@
+import TYPE from './command-type';
```

Comment:
It was my bad idea to split them into different files, let's use single file for all commands (we will not need separate server-side and client-side classes for this anymore)